packages/eslint-plugin/src/utils/nameIsImportedFromModel.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  ParserServices,
+  TSESTree,
+} from "@typescript-eslint/experimental-utils";
+import { AST_NODE_TYPES, TSNode } from "@typescript-eslint/typescript-estree";
+
+const matchModel = (name: string) =>
+  name.startsWith(".") && /model(\.ctx)?(\.(j|t)sx?)?$/.test(name);
+
+export const nameImportedFromModel = (
+  node: TSESTree.ImportDeclaration,
+  parserServices: ParserServices,
+  name: string,
+): {
+  namedImportNode: TSESTree.ImportSpecifier | undefined;
+  modelImportNode: TSESTree.ImportNamespaceSpecifier | undefined;
+} => {
```

Comment:
You can update this type, since we should always have exactly one as `undefined`.